stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/messages/list/PollMessageStyle.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.compose.ui.theme.messages.list
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import io.getstream.chat.android.compose.ui.theme.StreamColors
+import io.getstream.chat.android.compose.ui.theme.StreamShapes
+import io.getstream.chat.android.compose.ui.theme.StreamTypography
+
+/**
+ * Represents the style for poll messages.
+ *
+ * @param backgroundColor The background color for the poll message.
+ */
+public data class PollMessageStyle(
+    val backgroundColor: Color,
+) {
+
+    public companion object {
+
+        /**
+         * Builds the default poll message style for the current user's messages.
+         *
+         * Returns a [PollMessageStyle] instance holding our default theming.
+         */
+        @Suppress("DEPRECATION_ERROR")
+        @Composable
+        public fun defaultStyle(
+            own: Boolean,
+            isInDarkMode: Boolean,
+            typography: StreamTypography = StreamTypography.defaultTypography(),
```

Comment:
it's for further extension of the Style, to avoid changing the public API :)

Reply:
Just for my understanding: Is it OK to release a version of the SDK with such unused configurable arguments? I am worried that a user might expect that those customisations are possible, but in the end changing them would do nothing? Or is it expected to also extend the style before releasing the next version?

Reply:
I believe it should be fine. They’re there to future-proof the API, so we don’t have to break it when customization requests for polls come in. Most customers tweak `colors`, `typography` and `shapes` in the root `ChatTheme` anyway, so it’s more about keeping things flexible just in case.

Reply:
Ok, thanks for the explanation!